internal/handlers/irc/irc.go

```diff
@@ -17,11 +18,20 @@ type Client struct {
 	sendToTg func(string)
 }
 
+/*
+Creating variables for logging
+*/
+var (
+	logFlags    = log.Ldate | log.Ltime | log.Lmicroseconds | log.Llongfile
+	Info        = log.New(os.Stdout, "INFO: ", logFlags)
+	Error       = log.New(os.Stderr, "ERROR: ", logFlags)
```

Comment:
At the moment, this causes the code not to compile. You need to import `os` as well to use it's namespace. This also applies to telegram.go.

Reply:
Oh, also, since Go has syntactic capitalization, these variables are now both exported from this module, which is not desired. Please change these to `info` and `error`, respectively.